.travis.yml

```diff
@@ -10,6 +16,13 @@ matrix:
   exclude:
     - os: osx
       compiler: gcc
+before_install:
+        - apt-get source libnl-dev && cd libnl* && ./configure --prefix=$HOME && make && make install
```

Comment:
probably you want something like
- if [ "$TRAVIS_OS_NAME" = "linux" ]; then apt-get .... ; fi